python/py-toposort/Portfile

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-toposort
+version             1.7
+revision            0
+categories-append   devel
+license             Apache-2
+platforms           {darwin any}
+maintainers         nomaintainer
+description         Implements a topological sort algorithm
+long_description    {*}${description}
+homepage            https://gitlab.com/ericvsmith/toposort
+checksums           rmd160  370bc7d8af00509b3cfb269326856a5949becf3e \
+                    sha256  ddc2182c42912a440511bd7ff5d3e6a1cabc3accbc674a3258c8c41cbfbb2125 \
+                    size    10671
+supported_archs     noarch
+
+python.versions     36 37 38 39 310
```

Comment:
please don't add old Python versions (and certainly not EOL ones) anymore unless required by other packages. At this time I'd say only "310 311" are appropriate for new ports.

Reply:
@reneeotten Sure, i just included all versions listed on PyPi and confirmed to build.

Maybe keep 39 though? Not all Python packages support 310+, we can run into situation when a needed compo for a dependent does not exist.

Reply:
sure keeping py39 is fine. Python 3.10 has been the default this year and in January 1st it will be come 3.11

Reply:
@reneeotten Will be done. I will rebase the PR soon.

Reply:
@reneeotten Rebased. 311 not supported with this port, so kept 39 and 310: https://pypi.org/project/toposort/

Reply:
What do you mean by not supported - just that it isn't yet listed in the Trove classifiers by upstream or does the tests fail (if possible add the test phase so that these can be run to verify).

Reply:
Aren’t versions set by upstream?

Reply:
@reneeotten Ok, I will try.